vendor/github.com/giantswarm/k8sclient/Gopkg.toml

```diff
@@ -42,10 +42,6 @@
   branch = "master"
   name = "github.com/giantswarm/micrologger"
 
-[[constraint]]
-  branch = "master"
```

Comment:
I don't understand how it got deleted, we still use operatorkit by vendoring, don't we?

Reply:
This is in the vendor dir. I just ran `dep ensure -update github.com/giantswarm/k8sclient`

Maybe this is from the circular dependencies we removed.